cmd/all-in-one/main.go

```diff
@@ -130,11 +131,22 @@ by default uses only in-memory database.`,
 				HealthCheck:    svc.HC(),
 			})
 			c.Start(cOpts)
+			tCollectorOpts := new(tCollector.Options).InitFromViper(v)
+			tc := tCollector.Collector{}
+			tc.Start("jaeger-collector", tCollectorOpts, logger, c.SpanHandlers(), strategyStore)
```

Comment:
I would do this instead:
```suggestion
			tc := tCollector.New("jaeger-collector", tCollectorOpts, logger, c.SpanHandlers(), strategyStore)
```